src/components/pages/InstructorHome/InstructorProfile.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Card } from 'antd';
+import { Button } from 'antd';
+
+const { Meta } = Card;
+
+function InstructorProfile() {
+  return (
+    <Card
+      hoverable
```

Comment:
What is the attribute "hoverable" referring to? What aspect of the card component?

Reply:
"Hoverable" was there when I copied the entire card. It was having a Meta tag linked to instagram.com below that image,  if we put mouse on the card, it will show some shadow. I guess that is what hoverable is doing.  I was thinking to use a button linked to the instructor profile, maybe something like when we click, it will give a modal. So I changed the Meta tag to a button. Now, 'hoverable' seems to be not working. I can delete it. Do you think I need to make another pull request after I delete it?

Reply:
Sounds good. That makes sense. Lets play with this tag to see what kinds of effects we can get automatically from ant design.

Reply:
So you want me to change it back to Meta tag, or leave it like that for now and explore it later? Got to go now, but will check this later. Thank you for all your comments.